cirq/circuits/convert_to_cz_and_single_gates.py

```diff
@@ -50,47 +50,39 @@ def __init__(self,
         self.ignore_failures = ignore_failures
         self.allow_partial_czs = allow_partial_czs
 
-    def _convert_one(self, op: ops.Operation) -> ops.OP_TREE:
-        # Check if this is a CZ
-        # Only keep partial CZ gates if allow_partial_czs
-        if (isinstance(op, ops.GateOperation)
-                and isinstance(op.gate, ops.CZPowGate)
-                and (self.allow_partial_czs or op.gate.exponent == 1)):
-            return op
-
-        # Measurement?
-        if ops.MeasurementGate.is_measurement(op):
-            return op
-
-        # Known matrix?
-        mat = protocols.unitary(op, None)
-        if mat is not None and len(op.qubits) == 1:
-            return op
-        if mat is not None and len(op.qubits) == 2:
-            return decompositions.two_qubit_matrix_to_operations(
-                op.qubits[0],
-                op.qubits[1],
-                mat,
-                allow_partial_czs=False)
-
-        # Provides a decomposition?
-        decomposed = protocols.decompose_once(op, None)
-        if decomposed is not None:
-            return decomposed
-
-        # Just let it be?
-        if self.ignore_failures:
-            return op
-
-        raise TypeError("Don't know how to work with {!r}. "
-                        "It isn't composite or an operation with a "
-                        "known unitary effect on 1 or 2 qubits.".format(op))
-
     def convert(self, op: ops.Operation) -> ops.OP_TREE:
-        converted = self._convert_one(op)
-        if converted is op:
-            return converted
-        return [self.convert(e) for e in ops.flatten_op_tree(converted)]
+        def keep(op: ops.Operation) -> bool:
```

Comment:
Make this a private method on the class.

Reply:
Why this one in particular? 
Made all of them private again.

Reply:
I don't really have a hard and fast rule, it just seemed like it was about the class instead of the method specifically.